src/adv.py

```diff
@@ -67,16 +90,28 @@
 # If the user enters "q", quit the game.
 
 commands = ["n", "s", "e", "w"]
+print(player.room)
 
 while True:
-    print(player.room)
     command = input("\nSelect your move: ")
     if command == "q":
         print(f"\nGoodbye.\n")
         break
     elif command in commands:
         player.handle_command(command)
         player.room.get_exits()
+    elif command == "items":
+        player.room.list_items()
+    elif command == "grab":
+        if player.room.items is not None:
+            item = input("Item to pick up: ")
+            player.grab_item(item)
+        else:
+            print("No items in this room.")
+        # grab an item in current room
+    elif command == "drop":
+        player.room.list_items()
+        # drop an item from your inventory
     elif command == "help":
         print("\nNo help available right now, sorry bud.")
     else:
```

Comment:
probably another way to allow for player input without cluttering the main loop with if statements. could attempt to make that a function withing the player class and calling that function from the main loop.